dapp/config/default.json

```diff
@@ -83,5 +83,8 @@
   "debugTrace": true,
   "ipfsProxy": {
     "urlBase": "http://localhost:4000/api"
+  },
+  "dapp": {
```

Comment:
I think the default should use localhost, and this one is for QA environment. no?